package.json

```diff
@@ -1170,27 +1170,8 @@
     ]
   },
   "activationEvents": [
-    "onLanguage:ocaml",
-    "onLanguage:ocaml.interface",
-    "onLanguage:reason",
-    "onLanguage:ocaml.ocamllex",
-    "onLanguage:ocaml.menhir",
-    "onLanguage:dune",
-    "onLanguage:dune-project",
-    "onLanguage:dune-workspace",
     "onLanguage:cram",
```

Comment:
I think we can remove this too by adding a setting for cram to `breakpoints`.

Reply:
Looks like the way to get the warning to remove this line is to add a `languages/cram.json` file. I did that, but I'm not sure about the contents of that file 😅

Reply:
Ah, indeed, this change may be incorrect for cram...